elibrary-core/source/src/controller/java/com/javafee/elibrary/core/tabbedform/TabLoanServiceEvent.java

```diff
@@ -264,13 +264,17 @@ private void onClickBtnProlongation() {
 						JOptionPane.ERROR_MESSAGE);
 			}
 		} else
-			LogGuiException.logWarning(
-					SystemProperties.getInstance().getResourceBundle()
-							.getString("tabLoanEvent.notSelectedTablesWarningTitle"),
-					SystemProperties.getInstance().getResourceBundle()
-							.getString("tabLoanEvent.notSelectedTablesWarning"));
+			if (validatePenalty())
+				JOptionPane.showMessageDialog(tabbedForm.getFrame(),
+						SystemProperties.getInstance().getResourceBundle().getString("loanServicePanel.penaltyError") + " "
```

Comment:
Could u please use `MessageFormat` for this instead of String concatenation with '+' operator? 👀

Reply:
Done